src/DurableTask.Netherite/Connections/EventHubsUtil.cs

```diff
@@ -24,9 +24,9 @@ public static class EventHubsUtil
         /// <param name="eventHubName">The name of the event hub.</param>
         /// <param name="partitionCount">The number of partitions to create, if the event hub does not already exist.</param>
         /// <returns>true if the event hub was created.</returns>
-        public static async Task<bool> EnsureEventHubExistsAsync(string connectionString, string eventHubName, int partitionCount)
+        public static async Task<bool> EnsureEventHubExistsAsync(ConnectionInfo info, string eventHubName, int partitionCount, CancellationToken cancellationToken)
```

Comment:
This is technically a public API contract, but I get this was never meant for public use. Use your judgement here - do you make it separate methods to avoid breaking change or not?

In the next major version rev I definitely recommend evaluating public API surface and making all utilities like this and other implementation details classes `internal`.

Reply:
Yep, I was not particularly aware of how ANY public API surface becomes a liability. We can definitely retire some of these in the next major version change.